README.md

```diff
@@ -189,6 +189,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [gomod](https://github.com/camdencheek/tree-sitter-go-mod) (maintained by @camdencheek)
 - [x] [gowork](https://github.com/omertuc/tree-sitter-go-work) (maintained by @omertuc)
 - [x] [graphql](https://github.com/bkegley/tree-sitter-graphql) (maintained by @bkegley)
+- [ ] [hack](https://github.com/slackhq/tree-sitter-hack)
```

Comment:
should this get a checkmark now, that basic highlight queries are included?

Reply:
the checkmark depends on a maintainer being present. It's auto-generated.

Reply:
Ah, I see -- then we probably want to change the README (because that states the checkmark means "basically usable, has queries"; my bad!)

Since the maintainer information is already explicit, I'd prefer to use the checkmark to indicate independent information -- such as basic usability.

Reply:
sure, sounds reasonable.